spec/jobs/data_warehouse/daily_sensitive_column_report_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe DataWarehouse::DailySensitiveColumnJob do
+  subject(:job) { DataWarehouse::DailySensitiveColumnJob.new }
+
+  let(:timestamp) { Date.new(2024, 10, 15).in_time_zone('UTC') }
+  let(:job) { described_class.new }
+  let(:expected_bucket) { 'login-gov-analytics-export-test-1234-us-west-2' }
+  let(:tables) { ['auth_app_configurations'] }
+  let(:s3_data_warehouse_bucket_prefix) { 'login-gov-analytics-export' }
+
+  let(:expected_json) do
+    {
+      sensitive: [
+        {
+          "object-locator": {
+            "column-name": 'encrypted_otp_secret_key',
+            "table-name": 'auth_app_configurations',
+          },
+        },
+      ],
+      insensitive: [
+        {
+          "object-locator": {
+            "column-name": 'user_id',
+            "table-name": 'auth_app_configurations',
+          },
```

Comment:
1. where did "object-locator" terminology come from
2. what if we did one array of columns/fields with a `sensitive: true` boolean?

```
{
  fields: [
    {
      "column-name": "column-name",
      "table-name": "table-name",
      "sensitive": true,
    }
  ]
}
```

Reply:
we're getting the data structure from AWS DMS. We're creating the json file to match the json that the lambda will be checking against. This way the lambda can run the same lookup against the file and the DMS json.

Reply:
ooo that makes a lot more sense